org/postgresql/core/PGStream.java

```diff
@@ -135,17 +207,20 @@ public Encoding getEncoding() {
      */
     public void setEncoding(Encoding encoding) throws IOException {
         // Close down any old writer.
-        if (encodingWriter != null)
-            encodingWriter.close();
+        if (encodingWriter != null) {
```

Comment:
If you want to correct style, can you please send another PR for that?
It is very hard to identify changes required for unixsocket support.